glide.lock

```diff
@@ -32,6 +63,10 @@ imports:
   version: 1e291572dcb9ba673cdcdb15e0f422702415ebaf
 - name: github.com/satori/go.uuid
   version: 0aa62d5ddceb50dbcb909d790b5345affd3669b6
+- name: github.com/Sirupsen/logrus
```

Comment:
I don't see this in use else where in this PR. Was it just not added some other time when it should have been?